jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/ConfigsAsBooleanTest.java

```diff
@@ -13,41 +13,32 @@
  */
 package org.eclipse.jkube.kit.common;
 
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
 
-import java.util.Arrays;
-import java.util.Collection;
+import java.util.stream.Stream;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-@RunWith(Parameterized.class)
 public class ConfigsAsBooleanTest {
 
-  @Parameterized.Parameters(name = "{0}: asBoolean({1})={2}")
-  public static Collection<Object[]> data() {
-    return Arrays.asList(new Object[][] {
-        { "Unsupported String", " 1 2 1337", false },
-        { "One", "1", false },
-        { "Zero", "0", false },
-        { "FalseMixedCase", "fALsE", false },
-        { "False", "false", false },
-        { "True", "true", true },
-        { "TrueMixedCase", "TrUE", true },
-        { "TrueUpperCase", "TRUE", true }
-    });
+  @ParameterizedTest(name = "{0}: asBoolean({1})={2}")
+  @MethodSource("data")
+  void testUsingParametrizedTest(String testDesc, String input, Boolean expected) {
```

Comment:
testUsingParametrizedTest (sic) is not really telling anything don't you think?

Could we replace the method name with something more descriptive about what's tested instead of how it's tested, or at least the name of the method at test?